hack/make-rules/test-e2e-node.sh

```diff
@@ -141,8 +141,9 @@ if [ $remote = true ] ; then
   exit $?
 
 else
-  # Refresh sudo credentials if not running on GCE.
+  # Refresh sudo credentials for local run
```

Comment:
I think `if not running on GCE` is still valid. People may do development on GCE VM, and for that case we should not refresh sudo credential because GCE doesn't support that.